packages/default/scss/grid/_layout.scss

```diff
@@ -1221,6 +1221,32 @@
             }
         }
     }
+    .k-column-menu-tabbed {
+        border-radius: $kendo-tabstrip-item-border-radius $kendo-tabstrip-item-border-radius 0 0;
+
+        .k-tabstrip {
+            overflow: hidden;
+        }
+
+        .k-tabstrip-items {
+            margin: (-$kendo-tabstrip-item-border-width) (-$kendo-tabstrip-item-border-width) 0;
+
+            .k-item {
+                flex: 1;
+            }
+
+            .k-link {
+                height: $kendo-grid-column-menu-tabbed-tabstrip-item-height;
```

Comment:
I would suggest increasing the vertical padding, instead of adding variable for the height and setting a height. What if we increase the size of the icon, but the height is set

Reply:
We decided that overriding the tab height is not needed, therefore the code in questions was deleted.